R/rtabulate_refactor.R

```diff
@@ -34,7 +34,7 @@ setMethod("simple_analysis", "logical", function(x, ...) in_rows("Count" = rcell
 
 #' @rdname rtinner
 #' @exportMethod simple_analysis
-setMethod("simple_analysis", "factor", list_wrap_x(table))
+setMethod("simple_analysis", "factor", function(x, ...) in_rows(.list = as.list(table(x))))
```

Comment:
I wanted `simple_analysis` to return an `in_rows` object for consistency of type dispatch